CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Bug Fixes
 - Fix minor bugs for header slicing functionality and expand test coverage for edge-cases. (`#275 <https://github.com/DKISTDC/dkist/pull/275>`_)
 - Fixed inverse transform in `.VaryingCelestialTransformSlit2D`. Which fixes a bug in VISP WCSes. (`#285 <https://github.com/DKISTDC/dkist/pull/285>`_)
 - Fix a bug preventing the transfer of a single dataset with :meth:`~dkist.net.transfer_complete_datasets`. (`#288 <https://github.com/DKISTDC/dkist/pull/288>`_)
+- Update varying celestial transform schema ref to use a uri instead of a tag.(`#298 <https://github.com/DKISTDC/dkist/pull/298>`_)
```

Comment:
Same here as #297 could you add this to the `changelog/` directory please?

Reply:
Thanks! I added [298.bugfix.rst](b732ee713d38666f75c58435c20ccad2a697b2d0). Should this be a 'bugfix' or something else?